types/pino/index.d.ts

```diff
@@ -46,6 +46,7 @@ declare namespace P {
      */
     const LOG_VERSION: number;
     const levels: LevelMapping;
+    const symbols: { [key: string]: symbol };
```

Comment:
This is an improvement, but could we take it a step further and explicitly include the keys for the individual symbols? I copied the key names from [here](https://github.com/pinojs/pino/blob/db651a51762e8b043f406a73bb19470bcf6dcff1/lib/symbols.js#L37-L64).
```suggestion
    const symbols: {
        setLevelSym: symbol,
        getLevelSym: symbol,
        levelValSym: symbol,
        useLevelLabelsSym: symbol,
        mixinSym: symbol,
        lsCacheSym: symbol,
        chindingsSym: symbol,
        parsedChindingsSym: symbol,
        asJsonSym: symbol,
        writeSym: symbol,
        serializersSym: symbol,
        redactFmtSym: symbol,
        timeSym: symbol,
        timeSliceIndexSym: symbol,
        streamSym: symbol,
        stringifySym: symbol,
        stringifiersSym: symbol,
        endSym: symbol,
        formatOptsSym: symbol,
        messageKeySym: symbol,
        nestedKeySym: symbol,
        wildcardFirstSym: symbol,
        needsMetadataGsym: symbol,
        useOnlyCustomLevelsSym: symbol,
        formattersSym: symbol,
        hooksSym: symbol,
    };
```

Reply:
Great, thanks :)